gateway-server/src/main/java/org/apache/knox/gateway/services/DefaultGatewayServices.java

```diff
@@ -162,6 +162,8 @@ public void start() throws ServiceLifecycleException {
     SSLService ssl = (SSLService) services.get(SSL_SERVICE);
     ssl.start();
 
+    (services.get(TOKEN_SERVICE)).start();
```

Comment:
If we are going to add this here, we should add the corresponding `stop()` entry.

Reply:
agreed